.eslintrc

```diff
@@ -0,0 +1,158 @@
+rules:
+  indent:
+    - 2
+    - 2
+  quotes:
+    - 2
+    - single
+  linebreak-style:
+    - 2
+    - unix
+  semi:
+    - 2
+    - never
+  array-bracket-spacing: 
+    - 2
+    - "never"
+  block-scoped-var: 2
+  brace-style: 
+    - 2
+    - "stroustrup"
+    - 
+      allowSingleLine: true
+  comma-dangle: 
+    - 2
+    - "never"
```

Comment:
@tizzo @zanchin what do you two think about adding:
[vars-on-top: 2](http://eslint.org/docs/rules/vars-on-top.html) - makes it easy to see hoisted vars, big benefit for newer devs too

Also, I am very opposed to allowing comma-dangle because it's not valid all the time (e.g. in browsers, JSON, etc). I'd rather not allow something that can cause broken code. 

I could go either way with comma-style ... although commas first works really well with vars-on-top. Only thing with comma first is it always leads to a final line with only a semicolon, paren, or bracket ... kind of annoying, but nbd. [Easier to read diffs](https://gist.github.com/isaacs/357981#gistcomment-400) is kind of a neat point too ... I find it super ugly, but am somehow swayed to accepting it :)

What I do like about issac's style is a new line per object property and multi-dimensional arrays. But I care less for spacings between square and curly brackets.

About semicolons ... we usually minify code that runs in the browser, and I'm not sure ASI works without them in a minified state. We should write coffeescript if we hate semicolon use. I'm fine with that too.

```
var foo = bar
if (foo === bar) {
  var baz = 2
}
```
```
var foo = bar if (foo === bar) { var baz = 2 }
```

Last thing - is there a rule for a chaining method dot location? 

```
var saved = r
                   .get("123123123")
                   .update({ name: "New name" })
                   .run();
```
I think it's easier to read with the chained method on a new line ...